iwf/workflow_state_options.py

```diff
@@ -42,8 +46,16 @@ def _to_idl_state_options(
         return res
     assert isinstance(options, WorkflowStateOptions)
 
+    if options.wait_until_api_search_attributes_loading_policy is not None:
+        res.wait_until_api_search_attributes_loading_policy = options.wait_until_api_search_attributes_loading_policy
+    if options.execute_api_search_attributes_loading_policy is not None:
+        res.execute_api_search_attributes_loading_policy = options.execute_api_search_attributes_loading_policy
     if options.search_attributes_loading_policy is not None:
         res.search_attributes_loading_policy = options.search_attributes_loading_policy
+    if options.wait_until_api_data_attributes_loading_policy is not None:
+        res.wait_until_api_data_attributes_loading_policy = options.wait_until_api_data_attributes_loading_policy
+    if options.execute_api_data_attributes_loading_policy is not None:
+        res.execute_api_data_attributes_loading_policy = options.execute_api_data_attributes_loading_policy
```

Comment:
I had to manually add code to this file `iwf/workflow_state_options.py`
Everything else was generated after updating `iwf-idl`